pandas/io/formats/style.py

```diff
@@ -178,6 +178,23 @@ def to_excel(self, excel_writer, sheet_name='Sheet1', na_rep='',
                         startcol=startcol, freeze_panes=freeze_panes,
                         engine=engine)
 
+    def to_html(self, buf=None, mode='w', encoding='utf8'):
```

Comment:
The default encoding should probably match `to_csv`'s default of 

- ascii on python2
- utf-8 on python3

And you'll need to add it to the docstring.